components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/config/OAuthServerConfiguration.java

```diff
@@ -367,6 +369,12 @@ private void buildOAuthServerConfiguration() {
         parseHashAlgorithm(oauthElem);
         // read hash mode config
         parseEnableHashMode(oauthElem);
+
+        // Parse retain Access Tokens config.
```

Comment:
Can you please provide a more descriptive detail on this property? What is done when retain access token config is enabled?